src/styles/blocks/header-nav.scss

```diff
@@ -0,0 +1,58 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+
+    &__item {
+      text-align: center;
+      display: flex;
```

Comment:
The `display: flex;` property might not be necessary for the `&__item` if it doesn't contain any children that need to be flex items. Ensure that this is required for your layout.